code/modules/events/portal_storm.dm

```diff
@@ -21,6 +21,45 @@
 	hostile_types = list(/mob/living/simple_animal/hostile/construct/armored/hostile = 8,\
 						/mob/living/simple_animal/hostile/construct/wraith/hostile = 6)
 
+/datum/round_event_control/portal_storm_mesa
+	name = "Portal Storm: Mesa"
+	typepath = /datum/round_event/portal_storm/portal_storm_mesa
+	weight = 0
+	max_occurrences = 0
+
+/datum/round_event/portal_storm/portal_storm_mesa
+	boss_types = list(/mob/living/simple_animal/hostile/jungle/leaper = 4, \
```

Comment:
what about, the, ah, the creatures with flowers on their heads that shoot laser beams? should we add them to this list?

Reply:
what

Reply:
He means the Seedling.

Reply:
Uhhh maybe, not too sure on what the mob is like